db/migrate/20200223173254_remove_column_from_item_orders.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveColumnFromItemOrders < ActiveRecord::Migration[5.1]
+  def change
+    add_column :item_orders, :fulfilled?, :boolean, default: false
```

Comment:
This is entitled remove column.  Also the column is adds is already there with an enum